packages/docs-mdx/package.json

```diff
@@ -4,10 +4,10 @@
   "version": "0.1.3",
   "author": "HashiCorp",
   "dependencies": {
-    "@mdx-js/react": "1.6.22",
-    "@hashicorp/react-code-block": "^4.1.0",
+    "@hashicorp/react-code-block": "^4.4.2",
```

Comment:
Bumps `@hashicorp/react-code-block` to bring in manual copy fix from https://github.com/hashicorp/react-components/pull/503. Also makes an auto-formatting change for alphabetical ordering (moves `@mdx-js/react` to bottom of list).